assets/templates/signup.html

```diff
@@ -1,155 +1,224 @@
+                <input type="checkbox" name="acepto_politicas">
+                <span class="checkmark"></span><label for="acepto_politicas"> Acepto la <a class="link-green" href="#">Política de Privacidad</a> de Doppler.</label>
+                <div class="wrapper-errors">
+                  <p class="error-message">¡Ouch! Este campo está vacío</p>
+                </div>
+              </li>
+              <li class="field-item field-item__checkbox">
+                <input type="checkbox" name="acepto_promociones">
+                <span class="checkmark"></span>
+                <label for="acepto_promociones"> Acepto recibir las promociones de Doppler y sus aliados.</label>
+              </li>
+            </ul>
+            <button type="button" class="dp-button button--round button-medium primary-green">Crear cuenta</button>
+          </fieldset><!-- privacy policy and promotion -->
+        </form><!-- form -->
+        <div class="content-legal">
+          <p>Doppler te informa que los datos de carácter personal que nos proporciones al rellenar el presente formulario serán tratados por Making Sense LLC (Doppler) como responsable de esta web.</p>
+          <p><strong>Finalidad:</strong> Darte de alta en nuestra plataforma y brindarte los servicios que nos requieras.</p>
+          <p><strong>Legitimación:</strong> Consentimiento del interesado.</p>
+          <p><strong>Destinatarios:</strong> Tus datos serán guardados por Doppler, Zoho como CRM, Digital Ocean, Cogeco Peer1 y Rackspace como empresas de hosting.</p>
+          <p><strong>Información adicional:</strong> En la <a href="#" class="link-green" target="_blank">Política de Privacidad</a> de Doppler encontrarás información adicional sobre la recopilación y el uso de su información personal por parte de
+            Doppler, incluida información sobre acceso, conservación, rectificación, eliminación, seguridad, transferencias transfronterizas y otros temas.</p>
+        </div><!-- legal content-->
+        <p class="content-promotion">Pst! si tienes un código promocional podrás ingresarlo al confirmar tu cuenta. <a href="#" class="link-green" target="_blank">HELP</a></p>
+        <footer>
+          <small>© 2019 Doppler LLC. Todos los derechos reservados.</small>
+        </footer><!-- footer -->
+      </article>
+      <section class="feature-panel">
+        <article class="feature-content">
+          <h6>Editor de emails</h6>
+          <h3>Crea Emails en minutos y accede a nuestra Galería de Plantillas</h3>
+          <p>Nuestras Plantillas para Email son totalmente Responsive y fácilmente editables desde nuestro Editor HTML.</p>
+        </article><!-- article -->
+      </section><!-- feature panel-->
+    </main>
+    <!-- build:js:dist js/app.js -->
+    <link rel="stylesheet" type="text/css" href="https://intl-tel-input.com/node_modules/intl-tel-input/build/css/intlTelInput.css">
+    <style> .intl-tel-input.separate-dial-code .selected-dial-code {
```

Comment:
What are these styles?

Reply:
first there is the call of the style sheet of the library intl-tel-input, and then I overwrite two classes to be able to modify details.

Reply:
Solo para confirmar, ¿no deberíamos incluir esos estilos que sobreescribimos como parte de la librería? 

No estoy haciendo una apreciación, solo preguntando.

Reply:
Mmmm, yo creo que no. Si esto se incluye como un componente react seguramente tendremos que incluir los estilos dentro del componente y ahí darle los estilos correctos y finales.
Yo creo que si lo metemos en la librería, deberíamos meter TODO incluso los js.